.github/workflows/build-kata-static-tarball-s390x.yaml

```diff
@@ -31,6 +32,14 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
           fetch-depth: 0 # This is needed in order to keep the commit ids history
+
+      - name: Place a host key document
+        run: |
+          mkdir -p "host-key-document"
+          cp "${CI_HKD_PATH}" "host-key-document"
+        env:
+          CI_HKD_PATH: ${{ secrets.CI_HKD_PATH }}
```

Comment:
So the HKD is specific to the machine where the image will be run, correct? For the CI, we have a secret that stores an HKD that matches the runner where we test things. What about the image that is deployed via the operator? Can this tooling provide a valid image built ahead of time or does the client have to build their own?

Reply:
> So the HKD is specific to the machine where the image will be run, correct? For the CI, we have a secret that stores an HKD that matches the runner where we test things.

yes, correct.

> What about the image that is deployed via the operator? Can this tooling provide a valid image built ahead of time or does the client have to build their own?

ATM, a secure image in the operator is built only for a CI purpose. If a user wants to build an image and deploy it via the operator, he/she should provide his/her authenticated/authorized image registry (not sure if it is configurable on `main`) and an env variable `CI_HKD_PATH` where his/her own HKD is placed, which is bound to a machine to be deployed. 

Yes, this PR takes only a CI case into account.

Reply:
Ok, makes sense. Do we have a guide for clients who are building their own images?

Reply:
That is a plan discussed with Hendrik today :wink:

Reply:
FYI: https://github.com/kata-containers/kata-containers/pull/7146 will cover a client's needs.